### ISSUE TITLE
add_picture fails to dead with pictures in header

### ISSUE THREAD

**Issue by iohazard** [python-openxml/python-docx#356]
When using a template that has an image in the header. You add a picture it produces a document with an error that word will offer to fix. The fix involves renumbering the picture boxes document wide. It appears add_picture does not check the header before choosing the picture box number.

**Comment by scanny** [comment]
Please send an example document, before and after. Also, what version are you using?

**Comment by iohazard** [comment]
I am using 0.8.6 with a recent install of Anaconda Python 2.7 and Word 2013. In the attached zip is:

1. The script used to create the problem (word_equations.py) 
2. The files used as the template (test_input.docx)
3. The file output from the script (test_output.docx)
4. The file words generates when it fixes the file (test_output_fixed.docx)

[header_pic_issue.zip](https://github.com/python-openxml/python-docx/files/743791/header_pic_issue.zip)

**Comment by scanny** [comment]
Hmm, the file `test_output.docx` is loading just fine on my side. I'm running Mac Word 2011.

What do you mean when you say 'fix involves renumbering the pictures boxes'? I'm not aware of any picture boxes per se and what numbering they might have.

Is there a specific message or diagnostic information given when it doesn't load for you?

I see that the document is more complicated than most I've run across in that it includes a glossary (like a second embedded document) and some embedded XML for a bibliography or something. If you don't need that, you might try creating a new template document from scratch with your header and stuff in there, avoiding copying and pasting in Word itself (so just building it up item by item, inserting the picture in the header, trying it every so often to see what makes it fail or if it doesn't). If that works, that might be the quickest path to a fix for you :)

**Comment by iohazard** [comment]
I think the best quick fix falls in the category of "Why didn't I try that sooner?" I took the output that caused the error, allowed word to fix it, deleted the new content added by python and used that as the template and the result is in a clean file. So that is the fix I'll be using.

Thanks for the help! This is a great library I'm looking forward to using it more. In the interest of documenting the problem in case it helps with a more robust engine my debugging results are below:

I redid the document from a scratch using the built in word 2013 template and style. I added the letter head furniture clean without any copy and paste to make as clean a file as possible, it should not be a complicated document from a user standpoint. It appears that the addition of the word art vertical lines is the addition that breaks things. Using empty1.docx has the problem as before, empty1_nolines.docx does not. I also added the error box to the archive but it is not very useful.

What I meant by 'fix involves renumbering picture boxes' is that when I compared the header2.xml files between the broken and fixed word files in the original upload (using https://www.corefiling.com/opensource/xmldiff.html) the differences all looked like:

```
> -               <wp:docPr id='1' name='Text Box 1' />
> +               <wp:docPr id='3' name='Text Box 3' />
```

However in the later versions of the error this was not repeated so I withdraw that assertion.

[empty1_output.zip](https://github.com/python-openxml/python-docx/files/745094/empty1_output.zip)
